SPECS/vault.spec

```diff
@@ -1,13 +1,13 @@
 Name:           vault
-Version:        0.1.2
+Version:        0.6.1
 Release:        3%{?dist}
```

Comment:
Since I am really not an expert on SPEC Files, would you mind explaining what any number means before the percent sign?

(will update the request on monday, have the stuff on my work computer)